pkg/login/ldap_user.go

```diff
@@ -15,9 +19,7 @@ func (u *LdapUserInfo) isMemberOf(group string) bool {
 	}
 
 	for _, member := range u.MemberOf {
-		if member == group {
-			return true
-		}
+		return strings.EqualFold(member, group)
```

Comment:
Not sure this is correct. 

If the user is part of two groups it will always return based on the match of the first group. I don't think that's what you want.

Reply:
You are right, I'll update the PR.